media/js/privacy/privacy-quantum-firefox.js

```diff
@@ -0,0 +1,55 @@
+(function() {
+    'use strict';
+
+    if (document.querySelectorAll) {
```

Comment:
This should be `'querySelectorAll' in document`. Instead of wrapping it around the variable declarations here however, it may be simpler/neater to write a small function like:

```javascript
function supportsBaselineJS() {
    return 'querySelectorAll' in document &&
           'addEventListener' in window;
}
```

You could then use it like:

```javascript
if (supportsBasicJS()) {
    // execute page JS here
}
```

You could also apply this to the other JS file.

Reply:
I think I might have it right this time... Schalk wrote all this originally so I'm just butchering his stuff without really knowing what I'm doing 😟